src/js/lib/System.js

```diff
@@ -2,6 +2,14 @@
 
 import {remote} from "electron"
 
+let test = true
+
 export function showContextMenu(template: Object) {
-  new remote.Menu.buildFromTemplate(template).popup()
+  if (test) {
```

Comment:
@mikesbrown is there a way for the app to know that it's being run in an integration test?